sled-agent/src/illumos/dladm.rs

```diff
@@ -27,7 +27,7 @@ pub enum Error {
 }
 
 /// The name of a physical datalink.
-#[derive(Debug, Clone, Deserialize, Serialize)]
+#[derive(Debug, Clone, Deserialize, Serialize, PartialEq)]
```

Comment:
Not related to this PR, but I think this type should probably not be `Clone`.

Reply:
I kinda disagree - this type doesn't represent the physical link itself, it's just a strongly-typed name. Shouldn't names be cloneable?

Reply:
That's fair -- I think I'm conflating the case where it's a created device like a VNIC, which it clearly differs from. Ignore :)